src/limb_repo/controllers/base_controller.py

```diff
@@ -0,0 +1,31 @@
+"""Abstract Base Controller."""
```

Comment:
is this file incomplete? it looks like the classes are related to env stuff, not controller stuff

Reply:
Ah yeah I forgot to write this. I'm also wondering if you think this should be called a controller or planner?

Reply:
I'm not actually sure what you're planning to put here. but if it's the MPC stuff then I would call that a planner